shared/desktop/remote/component-loader.desktop.tsx

```diff
@@ -97,7 +97,7 @@ export default function(options: {
       <RemoteComponentLoader
         name={options.name}
         params={options.params || ''}
-        style={options.style}
+        style={options.style || null}
```

Comment:
why null instead of undefined here?

Reply:
can be, i didn't update all instances of the null vs undefined thing but we should